### ISSUE TITLE
Can i add colour when someone speaks something?

### ISSUE THREAD

**Issue by lukeranon** [morgolon/glife#16]
or should i just leave as it is?

also can i make changes in the images dir?
 like if :
picrand = 1:'<img src="images/qwest/ohrannik/bj1.jpg>"
picrand = 2:'<img src="images/qwest/ohrannik/bj2.jpg>
 to
                   <img src="images/qwest/ohrannik/bj<<picrand>>.jpg>" or this fucks things up?

**Comment by morgolon** [comment]
I think adding color will not be a problem, just follow how it's done elsewhere (I think it's standard html).

I'm going to test the second suggestion out and see what I find, it would make the code a little nicer to read :)

**Comment by heromaster** [comment]
I was thinking about that too to reflect an ongoing dialogue between two or more persons.

What would you suggest to use? We could also play with colors together with bolding, italic or something like that.

I already take care of the if picrand = x problem too and I'd say go ahead since I already have done the same :+1:

**Comment by lukeranon** [comment]
i really don't know,i just had the same idea that you have,but i am open to suggestions

**Comment by morgolon** [comment]
Yep I just looked at the code and templating image names like you suggest is done all over the source, so I'm sure it works just fine.

**Comment by heromaster** [comment]
I think the biggest challenge is to streamline those dialoguesm they are that are a terrible mix and match of all kind of styles.

Some examples:
"my dialogue", said MrX

or

Mrx: "my dialogue"

or just 'While the char speaking, I switch the POV of the narrator'
